app/assets/stylesheets/events.scss

```diff
@@ -16,3 +16,9 @@
   height: 300px;
   overflow: auto;
 }
+
+// Styling for Kaminari pagination
+span.page, span.next, span.last {
```

Comment:
something really tries to smash these together, so we expand them a little bit with margin.